apps/cross-components/rollup.config.lib.js

```diff
@@ -0,0 +1,124 @@
+/**
+ * Core Library Build
+ *
+ * This is the main rollup build for the Core library.
+ * The build compiles and optimizes the library for production.
+ */
+
+import * as fs from 'fs-extra';
+import autoprefixer from 'autoprefixer';
+import typescript from '@rollup/plugin-typescript';
+import nodeResolve from '@rollup/plugin-node-resolve';
+import replace from '@rollup/plugin-replace';
+import copy from 'rollup-plugin-copy';
+import del from 'rollup-plugin-delete';
+import minifyHTML from 'rollup-plugin-minify-html-literals';
+import styles from 'rollup-plugin-styles';
+import { terser } from 'rollup-plugin-terser';
+import {
+  packageCheck,
+  webComponentAnalyer,
+  addComponentEntryPoints,
+  createLibraryEntryPoints,
+  litSass,
+  globalStyles,
+  esmCache,
+} from './rollup.utils.js';
+
+/**
+ * This config defines your entry points for your library.
+ *
+ * Modules: entrypoints that are not components such as internal utilities
+ * Components: entrypoints for each component
+ * Styles: entrypoints for any global stylesheets to include with your library
+ * Assets: any static assets to be published with your library
+ */
+const config = {
+  baseDir: './src',
+  outDir: './dist/lib',
+  entryPoints: {
+    modules: ['./src', './src/internal'],
+    components: ['./src/component-status'],
+    styles: ['./src/global.scss'],
+    assets: ['./README.md'],
+  },
+};
+
+const prod = !process.env.ROLLUP_WATCH;
+const version = fs.readJsonSync('./npm.json').version;
+
+export default [
+  ...globalStyles({
+    baseDir: config.baseDir,
+    outDir: config.outDir,
+    styles: config.entryPoints.styles,
+  }),
+  {
+    external: [
```

Comment:
@bdryanovski  After having this, does the dist bundle still includes `lit` and `@cds/core`?

Reply:
No, it does not include them ... there are not needed when you bundle it for `lib` - lib is designed to depend on external builder like webpack, rollup, angular and so on. So dependencies like lit and @cds/core will have to be resolve by the parent. 

In our case for example the website is the parent, so @cds/core and lit element are coming from it.